cache/modules/wc_org_cloudfront/waf.tf

```diff
@@ -150,6 +150,75 @@ resource "aws_wafv2_web_acl" "wc_org" {
     }
   }
 
+  // See: https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-baseline.html#aws-managed-rule-groups-baseline-crs
+  rule {
+    name     = "core-rule-group"
```

Comment:
I'm not attached to this suggestion at all but you could consider commonising the managed rules sort of like 
```hcl
locals {
  managed_rule_groups = {
    // link to docs
    core = {
      name = "AWSManagedRulesCommonRuleSet"
      priority = 4
    }
    // link to docs
    sqli = {
       name = "AWSManagedRulesSQLiRuleSet"
       priority = 5
    }
    // link to docs
    known-bad-inputs = {
      name = "AWSManagedRulesKnownBadInputsRuleSet"
      priority = 6
    }
  }
}

// ...

dynamic "rule" {
  for_each = local.managed_rule_groups
  content {
    name = "${rule.key}-rule-group"
    priority = rule.value.priority
 
    override_action {
      count {}
    }
    
    statement {
      managed_rule_group_statement {
        name = rule.value.name
        vendor_name = "AWS"
      }
    }

    visibility_config {
      cloudwatch_metrics_enabled = true
      sampled_requests_enabled = true
      metric_name = "weco-cloudfront-acl-${rule.key}-${var.namespace}"
    }
  }  
}
```

Reply:
As soon as I wrote it I changed my mind on this, it just adds complexity, ignore

Reply:
A few more of these and this could be the way to go though.